### PR TITLE
Updating tests to replace .slice and .dynamicslice files

### PR DESCRIPTION
**Pull request by LesaelR**

### CODE REVIEW

File: AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_gui_tests_2.py [hunk 1]
```diff
@@ -264,7 +264,7 @@ def test_assets_added_to_cache():
 
         # Run AP GUI and read the config file we just modified to pick up our scan folder
         # Pass in a pattern so we don't spend time processing unrelated folders
-        result, _ = asset_processor.gui_process(quitonidle=True, add_config_scan_folders=True,
+        result, _ = asset_processor.gui_process(add_config_scan_folders=True,
```
Comment (jackalbe) [inline AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_gui_tests_2.py:267]: just curious why "quitonidle=True" was causing issues... maybe just on Linux?

File: AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_batch_tests_2.py [hunk 2]
```diff
@@ -143,37 +143,38 @@ def test_AllSupportedPlatforms_DeleteBadAssets_BatchFailedJobsCleared(
...
-            corrupted_slice = original_slice.replace("Components", "WWWWWWWWWWWW", 1)
+            original_prefab = file.read()
+            # This creates an intentional error in the prefab file which corrupts it and makes the asset fail
+            corrupted_prefab = original_prefab.replace("{}", "WWWWWWWWWWWW", 1)
 
         with open(copied_asset, "w") as file:
-            file.write(corrupted_slice)
+            file.write(corrupted_prefab)
 
+        print(corrupted_prefab)
```
Comment (jackalbe) [inline AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_batch_tests_2.py:170]: left over "print()" for debugging?

**Review by jackalbe — APPROVED** [review]

**Review by AMZN-stankowi — APPROVED** [review]

**Review by chiyenteng — APPROVED** [review]

**Review by chiyenteng — COMMENTED** [review]
If possible, can you wait @AMZN-koppersr 's review? I've asked his help since he's familiar with prefab and AP both.

File: AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/test_AddSameAssetsDifferentNames_ShouldProcess/test_prefab - Copy.prefab [hunk 3]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (AMZN-koppersr) [inline AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/test_AddSameAssetsDifferentNames_ShouldProcess/test_prefab - Copy.prefab:1]: Is this a copy that was accidentally submitted?

**Review by AMZN-koppersr — APPROVED** [review]
Not entirely sure why there are a several prefabs that just have "WWWWWW" or "{}".

**Review by chiyenteng — CHANGES_REQUESTED** [review]
I am not quite familiar with AP tests. Is that normal that test slices like C1564073/test_slice.slice being converted to "WWWWWW"?

File: AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/test_AddSameAssetsDifferentNames_ShouldProcess/test_prefab - Copy.prefab (hunk 3 above)
Comment (LesaelR) [reply AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/test_AddSameAssetsDifferentNames_ShouldProcess/test_prefab - Copy.prefab:1]: Each test looks at a different test asset folder, I could update these to all point to a single file later on and limit the number of copies in the test asset folders. 

As for the "WWWWW" prefabs, it's to cause the test to generate an error, then correct the error by fixing the prefab by re-writing it into a working JSON file - Originally these used a corrupted slice to achieve the same results.

File: AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_bundler_batch_tests.py [hunk 4]
```diff
@@ -1101,32 +1102,35 @@ def test_WindowsAndMac_AssetListSkipOneOfTwoParents_SharedDependencyIsIncluded(s
...
+            addSeed="objects/cube_lod0.azlod",
+            skip="objects/cube_lod0_position0.azbuffer, objects/cube_lod0_tangent0.azbuffer,"
+                 "objects/cube_lod0_normal0.azbuffer, objects/cube_lod0_bitangent0.azbuffer,"
+                 "objects/cube_lod0_uv0.azbuffer"
         )
         assert os.path.isfile(bundler_batch_helper["asset_info_file_result"])
         assets_in_list = []
         for rel_path in bundler_batch_helper.get_asset_relative_paths(bundler_batch_helper["asset_info_file_result"]):
             assets_in_list.append(rel_path)
+        print(assets_in_list)
```
Comment (amzn-mike) [inline AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_bundler_batch_tests.py:1133]: Did you mean to leave this in?

File: AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/C1564073/test_prefab.prefab [hunk 5]
```diff
@@ -0,0 +1 @@
+WWWWWWWWWW
```
Comment (amzn-mike) [inline AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/C1564073/test_prefab.prefab:1]: I'm a bit confused what this asset is for.  There's one below that's labelled corrupted, with the same contents.  I also saw a test which was replacing {} with WWWWWWWW in a prefab.  Just wondering what this one is used for

**Review by amzn-mike — APPROVED** [review]

File: AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_bundler_batch_tests.py (hunk 4 above)
Comment (LesaelR) [reply AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_bundler_batch_tests.py:1133]: No I missed 2 print statements I need to remove

File: AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/C1564073/test_prefab.prefab (hunk 5 above)
Comment (LesaelR) [reply AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/C1564073/test_prefab.prefab:1]: Yeah there are repeated assets, I can go back and consolidate them.

Comment (amzn-mike) [reply AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/C1564073/test_prefab.prefab:1]: Not a big deal really, it makes sense these tests have their own set of assets.  I guess I'm just more surprised that we have several different tests that are working with an invalid asset.

**Review by chiyenteng — APPROVED** [review]
Please fix nits mentioned above from other people.

**Review by chiyenteng — APPROVED** [review]

**Review by amzn-mike — APPROVED** [review]

**Comment by chiyenteng** [comment]
- test_validateDirectPreloadDependency_Found
- test_validateNestedPreloadDependency_Found

Next, please delete slice relate files and replace slice related terms (like testa.dynamicslice) with prefab/spawnable for these two tests. Add comments if needed to prevent confusion. Create GHIs/internal tickets for these tests, and lastly update/add the GHIs/tickets ids to the reason of @pytest.mark.skip.

**Pull request merged**